src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -14,6 +14,15 @@ var extendFlat = require('../../../lib/extend').extendFlat;
 
 
 module.exports = {
+    visible: {
+        valType: 'boolean',
+        role: 'info',
+        description: [
```

Comment:
Good point. I blindly copied this definition in a rush since it broke gl3d plots entirely until it was present.